code/modules/martial_arts/brawling.dm

```diff
@@ -38,8 +38,7 @@
 		if((D.stat != DEAD) && prob(knockout_prob))
 			D.visible_message("<span class='danger'>[A] has knocked [D] out with a haymaker!</span>", \
 								"<span class='userdanger'>[A] has knocked [D] out with a haymaker!</span>")
-			D.apply_effect(10,WEAKEN,armor_block)
-			D.Weaken(5)
+			D.Weaken(10 SECONDS)
```

Comment:
This removes the potential for an even higher weaken, (albeit mitigated by armour block values)